swmm-toolkit/requirements.txt

```diff
@@ -0,0 +1,18 @@
+aenum==2.2.4
```

Comment:
Pretty sure the only third-party package required is `aenum`. Prefer keeping it that way for cleaner install.